packages/aws-client/tests/test-S3.js

```diff
@@ -323,3 +330,58 @@ test('headObject() will retry if the requested key does not exist', async (t) =>
 
   await t.notThrowsAsync(promisedHeadObject);
 });
+
+const testEncodeTags = (t, input, expected) =>
```

Comment:
While this does result in fewer lines of code, I'm concerned that it sacrifices readability. My preference would be to see this refactored as `test('description', (t) => test)`.